src/MCPClient/lib/clientScripts/validateFile.py

```diff
@@ -153,14 +153,18 @@ def _execute_rule_command(self, rule):
         # NOTE: this requires that the stdout of all validation FPR commands be
         # a dict (preferably a JSON object) with an ``eventOutcomeInformation``
         # boolean attribute.
-        if output.get('eventOutcomeInformation') != 'pass':
-            print('Command {} indicated failure with this output:\n\n{}'.format(
-                rule.command.description, stdout), file=sys.stderr)
+        if output.get('eventOutcomeInformation') == 'pass':
```

Comment:
Not for this CR, but I wonder if the `eventOutcomeInformation` should be controlled in the model, or a class that FPR rules could in future inherit from? - I see `passed` and `failed`, `pass` and `fail` in this script for different purposes, and so it reminds me that there is room for error when not using controlled variable/const names.

Reply:
Yes, you're right that this needs fixing. We need to define a consistent API for how FPR Commands and client scripts can behave.

The work that @marktriggs is working on (see https://github.com/artefactual/archivematica/issues/938#issuecomment-371034310) may be relevant here.